webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/InterpolatedValueCompiler.scala

```diff
@@ -422,6 +431,7 @@ class InterpolatedValueCompilerImpl extends InterpolatedValueCompiler {
       case NonRudderVar(s)    => s"$${${s}}"
       case NodeAccessor(path) => s"$${rudder.node.${path.mkString(".")}}"
       case Param(name)         => s"$${rudder.param.${name}}"
+      case RudderEngine(e, m, p) => s"[missing value for: $${rudder.engine[${e}${m.mkString("][")}](${p.render()})}]"
```

Comment:
Please add a comment "should not happen since a non expanded engine property should lead to an error" or something alike